memcached.c

```diff
@@ -1684,11 +1686,16 @@ enum store_item_type do_store_item(item *it, int comm, LIBEVENT_THREAD *t, const
                 do_store = true;
                 break;
             case NREAD_CAS:
-                // LRU expired
-                stored = NOT_FOUND;
-                pthread_mutex_lock(&t->stats.mutex);
-                t->stats.cas_misses++;
-                pthread_mutex_unlock(&t->stats.mutex);
+                // Item missing.
```

Comment:
The `cas_res = CAS_MISS` above can be removed. Doesn't look like cas_res is used inside or outside of that block.